src/Fabulous.XamarinForms/Views/Controls/Slider.fs

```diff
@@ -30,3 +49,42 @@ module SliderBuilders =
                 Slider.ValueChanged.WithValue(fun args -> onValueChanged args.NewValue |> box),
                 Slider.MinimumMaximum.WithValue(struct (min, max))
             )
+
+[<Extension>]
+type SliderModifiers =
+
+    /// <summary>Set the color of the maximumTrackColor.</summary>
+    /// <param name="light">The color of the text in the light theme.</param>
+    /// <param name="dark">The color of the text in the dark theme.</param>
+    [<Extension>]
+    static member inline maximumTrackColor(this: WidgetBuilder<'msg, #ISlider>, light: Color, ?dark: Color) =
+        this.AddScalar(Slider.MaximumTrackColor.WithValue(AppTheme.create light dark))
+
+    /// <summary>Set the color of the minimumTrackColor.</summary>
+    /// <param name="light">The color of the minimumTrackColor in the light theme.</param>
+    /// <param name="dark">The color of the minimumTrackColor in the dark theme.</param>
+    [<Extension>]
+    static member inline minimumTrackColor(this: WidgetBuilder<'msg, #ISlider>, light: Color, ?dark: Color) =
+        this.AddScalar(Slider.MinimumTrackColor.WithValue(AppTheme.create light dark))
+
+    /// <summary>Set the color of the thumbColor.</summary>
+    /// <param name="light">The color of the thumbColor in the light theme.</param>
+    /// <param name="dark">The color of the thumbColor in the dark theme.</param>
+    [<Extension>]
+    static member inline thumbColor(this: WidgetBuilder<'msg, #ISlider>, light: Color, ?dark: Color) =
+        this.AddScalar(Slider.ThumbColor.WithValue(AppTheme.create light dark))
+
+    /// <summary>Set the source of the thumbImage.</summary>
+    /// <param name="light">The source of the thumbImage in the light theme.</param>
+    /// <param name="dark">The source of the thumbImage in the dark theme.</param>
+    [<Extension>]
+    static member inline thumbImageSource(this: WidgetBuilder<'msg, #ISlider>, light: ImageSource, ?dark: ImageSource) =
```

Comment:
Could you add the other types as well? Path, Uri, Stream

Reply:
Done 😀